azure_functions_worker/dispatcher.py

```diff
@@ -399,6 +411,14 @@ async def _handle__invocation_request(self, req):
                     fi.return_type.binding_name, call_result,
                     pytype=fi.return_type.pytype)
 
+            # Execute after invocation callbacks
+            for callback in get_after_invocation_request_callbacks():
+                try:
+                    callback(context)
+                except Exception as ex:
+                    logger.warning(
```

Comment:
Same here.

Reply:
Yeah same, this is system log. For extension to expose the log to users, they need to define a new logger without the `azure_functions_worker` prefix.
```python
import logging

logger = logging.getLogger('SomeExtension')
logger.info('xxx')
```